systemd/manager.go

```diff
@@ -106,6 +106,13 @@ func (m *systemdUnitManager) Load(name string, u unit.UnitFile) error {
 	if err != nil {
 		return err
 	}
+	if u.HasReverseDependencies() {
```

Comment:
I think the idea was to try *always* enabling units on load? This kind of special-casing seems fragile, unnecessary, and confusing...